apps/hooks/pipeline/code-artifact-docker/index.ts

```diff
@@ -25,6 +23,8 @@ exports.handler = async (
   statusCode: number;
   body: string;
 }> => {
+  console.log(JSON.stringify(event));
```

Comment:
maybe remove this since it's for debugging purposes

Reply:
good catch :tropical_fish: